### ISSUE TITLE
Is this package abandonware?

### ISSUE THREAD

**Issue by jaytaylor** [sergi/go-diff#35]
Currently seeing many open PR's with little or no discussion from @sergi.

Do we need to fork this to get all the excellent updates merged?

**Comment by zimmski** [comment]
I talked with @sergi and he gave me contributor permissions on the repository. However, someone has to review my PRs. I am still waiting on his review so we can merge #34. After that I will review each open PR, and then adapt and merge them. #34 is needed because without a CI there would be no automatic testing new changes, and I do not want any surprises.

**Comment by jaytaylor** [comment]
Roger that, thanks for the info.

Leaving this open so others can easily find this out!

**Comment by zimmski** [comment]
I will close this issue now, since I have sufficient rights on the repository to move forward. However, I am in meetings for the next 2-3 days. I will start ASAP to work down PRs and issues. It is happening this week though since I still have booked hours :+1: 

@jaytaylor: If you have any changes please just summit a PR :-)

**Comment by zimmski** [comment]
Sidenote: This is a duplicate of #33
